CHANGELOG-1.3.md

```diff
@@ -0,0 +1,27 @@
+# Changelog since v1.2.0
```

Comment:
Do we want to compare against 1.2.0 (which was prematurely released), or 1.2.1?

Reply:
IMO we should follow what Kubernetes does and its 1.14 changelog starts with ` Changelog since v1.13.0`